model/sharing/member.go

```diff
@@ -462,9 +462,9 @@ func (s *Sharing) DelegateDiscovery(inst *instance.Instance, state, cozyURL stri
 	return success["redirect"], nil
 }
 
-// UpdateRecipients updates the list of recipients
-func (s *Sharing) UpdateRecipients(inst *instance.Instance, members []Member) error {
-	for i, m := range members {
+// UpdateRecipients updates the lists of members and groups.
+func (s *Sharing) UpdateRecipients(inst *instance.Instance, params PutRecipientsParams) error {
```

Comment:
I believe this is a breaking change and would prevent valid communication between stacks of different versions?!

Reply:
Well, "breaking change" is always a relative notion. But a new stack can still send the updated recipients list to an old stack, it is just that the groups will be ignored. And an old stack doesn't have any groups, and can send the recipients list to a new stack without issue.